dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/collection/CollectionUtils.java

```diff
@@ -402,4 +403,11 @@ public static <T> Map<T, T> mergeMaps(Map<T, T>... maps) {
     Stream.of(maps).forEach(result::putAll);
     return ImmutableMap.copyOf(result);
   }
+
+  public static <T extends BaseIdentifiableObject> List<String> toUidValueList(
```

Comment:
I am pretty sure we did have a utility like this in `UID`s class.

Reply:
we don't, this is the closest thing we have (obj array):
`public static Set<UID> of(@Nonnull UidObject... objects)`

I could move this into the `UID` class though if we want?

Reply:
Yeah, please do